ramls/calendar.raml

```diff
@@ -174,32 +174,17 @@ resourceTypes:
                 description: "Internal server error, e.g. due to misconfiguration"
 
       /calculateopening:
-        is: [language]
         get:
+          is: [language]
           queryParameters:
-            startDate:
-              displayName: startDate
+            requestedDate:
```

Comment:
@kostyantyn-kh I thought in [UICIRC-178](https://issues.folio.org/browse/UICIRC-178) your proposal was for date and time query parameters to use the [datetime type](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#date)?

Reply:
@marcjohnson-kint datetime (and datetime-only) type doesn't work as expected, we'll raise a ticket

Reply:
@kostyantyn-kh @opochepam  can you help me understand why this parameter needs a date and a time?

To me, this endpoint is for calculating the opening times of a service point on a few days, centred around a chosen date. Why is the time on the chosen date relevant?

Reply:
@marcjohnson-kint you are right, this parameter needed time in previous implementation when endpoint accepted start date, unit and amount and mod-calendar calculated requested date, but now requested date is calculated by mod-circulation so we do not need time anymore. I will commit changes soon.